composer.json

```diff
@@ -22,13 +22,13 @@
     },
     "require": {
         "php": ">=5.4.0",
-        "beatswitch/lock": "0.1.*"
+        "beatswitch/lock": "0.1.*",
+        "illuminate/auth": "5.0.x",
+        "illuminate/database": "5.0.x",
+        "illuminate/support": "5.0.x"
```

Comment:
I use `*` for most of my projects so preferable that one.

I think at one point someone posted an unofficial poll on twitter which was more used and I think the `*` beat the `x` by a vast majority. But I have no idea anymore who posted it or where the results are.